python_modules/dagster/dagster/_daemon/asset_daemon.py

```diff
@@ -22,7 +22,7 @@
 CURSOR_KEY = "ASSET_DAEMON_CURSOR"
 ASSET_DAEMON_PAUSED_KEY = "ASSET_DAEMON_PAUSED"
 
-EVALUATIONS_TTL_DAYS = 7
+EVALUATIONS_TTL_DAYS = 30
```

Comment:
I'll admit this is a very dumb comment, but should we technically make this 31 days?

Reply:
alt: change the description to say that they're retained for 30 days